### PR TITLE
fix: bug fixes + add missing kwargs

### PR DESCRIPTION
**Pull request by WiemKhlifi**
## What?
Set the appropriate type for `episode_return` in the `Checkpointer` class, fix the `ExperimentOutput` type in the quickstart notebook, and add missing kwargs in the `jumanji_make_env` function.
## Why?
The changes made have resolved the bugs identified.

### CODE REVIEW

File: examples/Quickstart.ipynb
Comment (OmaymaMahjoub) [inline examples/Quickstart.ipynb:1]: @WiemKhlifi  double check the rware version that you are using, as I can see from the rendering and results that you are using the old version of rware before the fix that was made in jumanji.

File: mava/utils/make_env.py [hunk 1]
```diff
@@ -72,10 +72,9 @@ def make_jumanji_env(
     wrapper = _jumanji_registry[env_name]["wrapper"]
 
     # Create envs.
-    env = jumanji.make(env_name, generator=generator)
-    env = wrapper(env)
+    env = jumanji.make(env_name, generator=generator, **config.env.kwargs)
```
Comment (OmaymaMahjoub) [inline mava/utils/make_env.py:75]: Nice catch!

**Review by OmaymaMahjoub — CHANGES_REQUESTED** [review]
Thanks @WiemKhlifi, if you can rerun the notebook with the correct rware version and from there happy to approve the PR

**Comment by WiemKhlifi** [comment]
All checks are done 🙌

File: mava/utils/logger.py [hunk 2]
```diff
@@ -393,18 +393,19 @@ def write(
...
-            step_metrics = {"step_count": timestep, "elapsed_time": current_time - self.start_time}
-            step_metrics.update(metrics)  # type: ignore
+        if logging_prefix == "absolute":
+            self.run_data["absolute_metrics"].update(metrics)
+
+        elif logging_prefix == "evaluator":
+            step_metrics = {
+                "step_count": timestep,
+                "elapsed_time": current_time - self.start_time,
+            } | metrics
```
Comment (sash-a) [inline mava/utils/logger.py:403]: I definitely change this in #994. Can we get that in first? And then make the change on top of that?

**Review by sash-a — COMMENTED** [review]
Nice, I like this just need to wait for #994 I think

File: mava/utils/logger.py (hunk 2 above)
Comment (sash-a) [reply mava/utils/logger.py:403]: Scratch that I was thinking of the other logging PR, so never mind this. Will approve :smile:

**Review by sash-a — APPROVED** [review]
:fire_engine: :fire:

**Review by OmaymaMahjoub — APPROVED** [review]
Thanks @WiemKhlifi  LGTM!

**Pull request merged**